### PR TITLE
api updates

### PR DESCRIPTION
**Pull request by catsoup11789**

### CODE REVIEW

File: code/aspen_app/App.js [hunk 1]
```diff
@@ -5,13 +5,19 @@ import * as Sentry from 'sentry-expo';
...
-	sessionTrackingIntervalMillis: 10000,
-	debug: true, // If `true`, Sentry will try to print out useful debugging information if something goes wrong with sending the event. Set it to `false` in production
-});
+import { LogBox } from 'react-native';
+//LogBox.ignoreLogs(['Warning: ...']); // Ignore log notification by message
+//LogBox.ignoreAllLogs();//Ignore all log notifications
+
+if (!__DEV__) {
+	Sentry.init({
+		dsn: 'https://7fe05c932e2b4fd7b93fa3a0acae0649@o994439.ingest.sentry.io/5952947',
```
Comment (mdnoble73) [inline code/aspen_app/App.js:14]: Is it ok for this to be in code?

File: code/web/services/API/UserAPI.php [hunk 2]
```diff
@@ -375,18 +375,42 @@ function getPatronProfile() : array
 			}
 
 			$numCheckedOut = 0;
+			$numOverdue = 0;
 			$numHolds = 0;
 			$numHoldsAvailable = 0;
 			$accountSummary = $user->getAccountSummary();
 			$userData->numCheckedOutIls = (int)$accountSummary->numCheckedOut;
 			$userData->numHoldsIls =(int) $accountSummary->getNumHolds();
-			$userData->numHoldsAvailableIls = (int) ($accountSummary->numAvailableHolds == null ? 0 : $accountSummary->numAvailableHolds);
```
Comment (mdnoble73) [inline code/web/services/API/UserAPI.php:383]: This was probably done for Arlington? Is casting it to an int causing an issue

File: code/web/services/API/UserAPI.php [hunk 3]
```diff
@@ -375,18 +375,42 @@ function getPatronProfile() : array
...
 			$numCheckedOut += $userData->numCheckedOutIls;
 			$numHolds += $userData->numHoldsIls;
 			$numHoldsAvailable += $userData->numHoldsAvailableIls;
+			$numOverdue += $userData->numOverdue;
+
+			$userData->expires = $accountSummary->expiresOn();
+			$userData->expireClose = $accountSummary->isExpirationClose();
+			$userData->expired = $accountSummary->isExpired();
+
+			if ($user->getLinkedUsers() != null) {
```
Comment (mdnoble73) [inline code/web/services/API/UserAPI.php:397]: We probably need a switch for whether or not to include Linked Users.  Arlington handles linked users in a different way in the app so we would want the default off.

File: code/web/services/API/UserAPI.php [hunk 4]
```diff
@@ -405,9 +429,23 @@ function getPatronProfile() : array
 				$userData->numCheckedOutOverDrive = (int)$overDriveSummary->numCheckedOut;
 				$userData->numHoldsOverDrive = (int)$overDriveSummary->getNumHolds();
 				$userData->numHoldsAvailableOverDrive = (int)$overDriveSummary->numAvailableHolds;
-				$numCheckedOut += $userData->numCheckedOutOverDrive;
-				$numHolds += $userData->numHoldsOverDrive;
-				$numHoldsAvailable += $userData->numHoldsAvailableOverDrive;
+				$numCheckedOut += (int)$overDriveSummary->numCheckedOut;
```
Comment (mdnoble73) [inline code/web/services/API/UserAPI.php:432]: Similar to above, let's add an option to either include linked user data or not

**Review by mdnoble73 — CHANGES_REQUESTED** [review]
Let's drop the files in .tmp and a few other comments below

File: code/aspen_app/App.js (hunk 1 above)
Comment (catsoup11789) [reply code/aspen_app/App.js:14]: Sentry says DSNs are safe to be in public code, but I'll be reworking this eventually to come from the config file to be able to send errors to different projects for branded apps.

File: code/web/services/API/UserAPI.php (hunk 2 above)
Comment (catsoup11789) [reply code/web/services/API/UserAPI.php:383]: (int) was greyed out in phpstorm, like it would never be accessed, but I can add it back in just in case.

**Review by mdnoble73 — APPROVED** [review]

**Pull request merged**